src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -146,8 +146,41 @@ public override ExportResult Export(in Batch<Metric> batch)
                         }
                         else
                         {
-                            // TODO: Consider how/if to display buckets for exponential histograms.
-                            bucketsBuilder.AppendLine("Buckets are not displayed for exponential histograms.");
+                            var exponentialHistogramData = metricPoint.GetExponentialHistogramData();
+                            var scale = exponentialHistogramData.Scale;
+                            var factor = Math.Log(2) * Math.Pow(2, -scale);
+
+                            var offset = exponentialHistogramData.NegativeBuckets.Offset;
+                            List<string> negativeBucketCounts = null;
+                            foreach (var bucketCount in exponentialHistogramData.NegativeBuckets)
+                            {
+                                if (negativeBucketCounts == null)
+                                {
+                                    negativeBucketCounts = new List<string>();
+                                }
+
+                                var lowerBound = Math.Exp(offset * factor).ToString("F6", CultureInfo.InvariantCulture);
+                                var upperBound = Math.Exp(++offset * factor).ToString("F6", CultureInfo.InvariantCulture);
+                                negativeBucketCounts.Add($"Bucket [-{lowerBound}, -{upperBound}), Count: {bucketCount}");
+                            }
+
+                            for (var i = 0; i < negativeBucketCounts.Count; ++i)
+                            {
+                                bucketsBuilder.AppendLine(negativeBucketCounts[negativeBucketCounts.Count - i - 1]);
+                            }
+
+                            if (exponentialHistogramData.ZeroCount != 0)
+                            {
+                                bucketsBuilder.AppendLine($"Bucket [0, 0], Count: {exponentialHistogramData.ZeroCount}");
```

Comment:
I guess ZeroCount is for `[0, IEEE754 epsilon / 2]`.

Reply:
From a learning perspective, I think the output helps folk derive an intuition for exponential histograms. I'm not opposed to getting more technical and surfacing up details of the underlying IEEE 754 representation of things, but I wonder if it's helpful.

Reply:
English works for me.